### ISSUE TITLE
Niche channels one may join if one so desires

### ISSUE THREAD

**Issue by Bentechy66** [CyberDiscovery/meta#381]
**Is your feature request related to a problem? Please describe.**
You know when you're chilling in general and some weird gambling lookin dude with a "wall street bets" t-shirt and aviator glasses comes in and posts some huge graph you don't care about then proceeds to boast about how their master investment strategy made then 86 pence in the last year? That sucks. Or when one of the absolute chads of the server drops in and starts a mountain biking conversation, but is interrupted by some other discussion starting.

**Describe the solution you'd like**
The clear solution here is to stop people treading on other's toes - and thus, I introduce to you a revolutionary new system - A collection of channels one can join with `!join #channel`, or leave if one is not interested in the subject matter and/or channel list clutter. Example channels include `#stonks`, `#cycling`, `#food` to name a few.

**Additional context**
seriously though please keep the gambling trading bois out of general, and before you ask, yes, this is a serious suggestion

**Comment by ghost** [comment]
I back this fully, I think its a good idea to keep channel clutter down while still giving people a place to really talk about what interests them without getting interrupted or over taking the chat. 

This does not mean that general should be void of these subjects. It would just be nice to have somewhere to have a more in depth conversation. 

Use overflow you say? Well the problem there is overflow is a spam channel, you cant scroll back and look at messages as it is full of random stuff. 

Use group chats you say? That doesn't really work either as group chats are static. Once you add some people you have to actively find more people to add if you want more people to talk to. If it was done through cd people can join channels they are interested in and they don't have to even really know the people in the group before joining it.

**Comment by woaisthisgit** [comment]
I think this is dumb. Just talk in general it's literally not that hard to follow a conversation you are interested in.

**Comment by ghost** [comment]
> I think this is dumb. Just talk in general it's literally not that hard to follow a conversation you are interested in.

That wasn't the point of this.

**Comment by lightspeedana** [comment]
I'm conflicted. I agree entirely with the sentiment but I also see no way to do this without cluttering the server with a bunch of channels, or having to make infinite channels because other people have niche interests that they'd like a channel for. The system would simply be too complex.

**Comment by ghost** [comment]
We can start with a few necessary channels- `#cycling` , which has been requested many times before, and `#stonks` , which is just needed. Neither of those are very niche as cycling is something lots of server members do, and stocks is clearly something a group engage in. It would be great to be able to share that topic with others who are interested rather than feeling like you clog up general.

**Comment by ghost** [comment]
> I'm conflicted. I agree entirely with the sentiment but I also see no way to do this without cluttering the server with a bunch of channels, or having to make infinite channels because other people have niche interests that they'd like a channel for. The system would simply be too complex. 

@lightspeedana I don't see where there would be clutter, they are joinable so will be hidden to most people. Of course staff will see them all, but then again iirc there are about twenty meme channels only viewable by staff. Its not complicated, if more than a few ask for a channel it can be made, ppl ask stuff all the time. And get denied. So nothing would change.

**Comment by woaisthisgit** [comment]
> > I think this is dumb. Just talk in general it's literally not that hard to follow a conversation you are interested in.
> 
> That wasn't the point of this.

What is the point then? Keep "clutter" down? I don't see why that's needed just let people talk about what they want to talk about, that's the point of a general channel. Plus I doubt people will use them for more than a week unless told to by staff

**Comment by R0w4nS** [comment]
#general is for general conversation; stonks, mountain biking and food all fit in that category

**Comment by ghost** [comment]
People in general aren't going to love people sending links and discussing about cycling, the industry etc. It's so there would be a more concentrated area, as Hackerjacker said. I don't think the people against it here properly understand that that's why the proposal was made.

**Comment by alfiejfs** [comment]
People have already explained that general would become less active so I don't need to explain that again.

What I will say is that this server is Cyber Discovery. Cyber Discovery does not and should not have a channel for the stock market. It makes no sense. Same applies for cycling, and whatever other random channels you propose. General, for a server of this nature, is for stuff that isn't Cyber Discovery which includes trading and cycling etc. The only exception is for the most broad and most frequently discussed stuff like gaming or tech or politics, but even then a lot of that discussion ends up in general anyway - so why convolute the server even more? Just come back when the topic is different.

**Comment by lightspeedana** [comment]
@cyberlil1 I really don't know how else to say this but, they're not necessary. We already have so many channels which are often hard to keep track of, and as @alfiejsmith said, it's Cyber Discovery. Even before I was a mod, group DMs were things that I'd make and engage in for such niche interests, and I really honestly don't see why you can't send friend requests to each other and click on "Create Group DM", and instead have to make it in a server that frankly isn't only for you - it's for 3000 other people as well.

**Comment by Bentechy66** [comment]
yo wtf

**Comment by 0xAda** [comment]
this is a fucking dumb idea

**Comment by cyberlil1** [comment]
david is a retard

**Comment by n3bb** [comment]
~~Stonks Discovery~~
